### ISSUE TITLE
Attribute alt not rendered in front end img tag in 11.0.10 and 11.0.11

### ISSUE THREAD

**Issue by abach42** [netresearch/t3x-rte_ckeditor_image#247]
Please take a look in 11.0.10

https://github.com/netresearch/t3x-rte_ckeditor_image/blob/b497633170b85d97af67ab766e373c5b5963d14d/Classes/Controller/ImageRenderingController.php#L266C14-L266C14

```
if (isset($attributes[$attributeNameOverride]) && isset($attributes[$attributeName])) {
            $attributeValue = $attributes[$attributeName];
```

In case `$attributes['data-alt-override']` does not exist, `$attributes['alt']` is not written to `$attributeValue`.

Bugfix Version 11.0.9 had this: 

``` 
if (isset($attributes[$attributeNameOverride])) {
            $attributeValue = $attributes[$attributeNameOverride];
} elseif (isset($attributes[$attributeName])) {
            $attributeValue = $attributes[$attributeName];
```
That was dead simple: if _a_ then use _a_, if _b_ then use _b_. 

So I downgraded to 11.0.9 to keep using extension. 
Please consider to use design patterns for business logic in this class.

**Comment by magicsunday** [comment]
Hi, what exactly is the problem now? Where does which error occur? How can this be recreated?

**Comment by DavidBruchmann** [comment]
The problem seems to be that `$attributes[$attributeNameOverride]` is not take into consideration currently.

**Comment by DavidBruchmann** [comment]
Instead `$attributes[$attributeNameOverride]` and `$attributes[$attributeName]` have to exists both that `$attributeValue` gets assignend.  
It's actually a strange logic currently.

**Comment by magicsunday** [comment]
The question is, does misbehavior occur somewhere that deviates from the expected behavior and if so, how can I recreate it? I couldn't find anything during my test. Instead, it seems to me that checking `$attributeNameOverride` is not necessary at all at this point (only in the JS to toggle the checkbox state). Since the "alt" or "title" attribute apparently always contains a value and `data-alt-override` assumes either `true` or `false`.

So the check should be removed I think.

**Comment by Gitsko** [comment]
Should be fixed in the newest version (v11.0.14)
